src/coreComponents/unitTests/constitutiveTests/CMakeLists.txt

```diff
@@ -46,3 +51,21 @@ foreach(test ${gtest_geosx_tests})
                 )
 
 endforeach()
+
+
+#
+# Add triaxial xml based tests
+#
+
+blt_add_executable( NAME testTriaxial
+                    SOURCES testTriaxial.cpp
+                    OUTPUT_DIR ${TEST_OUTPUT_DIRECTORY}
+                    DEPENDS_ON ${dependencyList} "-lz"
+                  )
+
+foreach(test ${gtest_triaxial_xmls})
+    get_filename_component( test_name ${test} NAME_WE )
+    blt_add_test( NAME ${test_name} 
+                  COMMAND testTriaxial -i ${CMAKE_CURRENT_LIST_DIR}/${test}
```

Comment:
This is more of an integrated test than a unit test? But I guess you can't perform an integratedTest without a mesh?